src/Container/NotFoundMiddlewareFactory.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
```

Comment:
Remove empty line before strict_type declaration.

Reply:
This is not relevant at this time _as all other files in this branch already have an empty line between the file-level docblock and the declaration_; as such, this is just internally consistent.

We need to have a separate discussion in the zend-coding-standard repo about what our standard will be, as we now have two repos using two different approaches.

Reply:
@weierophinney actually I was wrong, sorry, please see my comment there:
https://github.com/zendframework/zend-expressive-aurarouter/pull/30#discussion_r164587191